test/parser/test_parser.py

```diff
@@ -583,3 +583,30 @@ def test_dump_email() -> None:
     output_data = validator.validate(data, base_url="http://example.com")
     assert output_data == data
     assert isinstance(output_data["email"], str)
+
+
+def test_computed_field() -> None:
+    schema = {
+        "degree": {"type": "string"},
+        "first_name": {"type": "string"},
+        "last_name": {"type": "string"},
+        "full_name": {
+            "type": "string",
+            "expression": "CONCAT(first_name, ' ', last_name)",
```

Comment:
we'd also need to parse this stuff in the front end which might be annoying. Should we make this an object?

Reply:
```suggestion
            "expression": {
                "name": "CONCAT"
                "params": [
                        {"name": "CONCAT", "params": ["'test'", " ", "'zest'"]},
                        "field.other_field"
                .........
```

Reply:
Decided to leave as string so the users can "program"